konnectivity-socks5-proxy/main.go

```diff
@@ -139,15 +159,46 @@ func dialFunc(caCertPath string, clientCertPath string, clientKeyPath string, pr
 	}
 }
 
-// dialDirect directly connect directly to the target, respecting any local proxy settings from the environment
-func dialDirect(ctx context.Context, network, addr string) (net.Conn, error) {
+// dialDirectWithoutProxy directly connect to the target, ignoring any local proxy settings from the environment
+func dialDirectWithoutProxy(ctx context.Context, network, addr string, dnsFallbackToMC *dnsFallbackToManagementCluster) (net.Conn, error) {
+	var d = net.Dialer{
+		Timeout: 2 * time.Minute,
+	}
+	connection, err := d.DialContext(ctx, network, addr)
+	if err != nil {
+		return nil, err
+	}
+	dnsFallbackToMC.set(false)
```

Comment:
We should not set it here. This should change as soon as we can validly resolve with the cluster's nodes and never change back.